### DISCUSSION TITLE
[Projects beta] URLs in custom text field doesn't work as a link in issue pages.

### DISCUSSION THREAD

**Discussion by hiragram** [community/community#12901] [Projects and Issues]
I am adding a URL for in-house Notion page as a custom text field.
It appears in the issue page, but clicking the URL doesn't navigate to the page and start editing its value. This is inconvenient.

<img width="314" alt="スクリーンショット 2022-03-16 5 46 13" src="https://user-images.githubusercontent.com/3433324/158469796-cf9d97f3-ba88-455c-b44e-d036f2c3ceee.png">

**Comment by decatoncheir** [comment]
The custom text field in issue pages now has a pen icon for field value edit.

However, when clicking the url text still edit the field

Expected behaviour: click the url and go to the linked url.

**Comment by dominikklein** [comment]
Any news about this? Very frustrating that a URL can not really be used in a text field or that there is no URL field type.
